src/logger.ts

```diff
@@ -0,0 +1,57 @@
+const winston = require("winston");
+
+// list of logging levels in order
+export const logLevels = ["error", "warn", "info", "verbose", "debug", "silly"];
+
+// read in environment variables
+let logLevel = process.env.RAPTOR_LOG_LEVEL && logLevels.includes(process.env.RAPTOR_LOG_LEVEL) ? process.env.RAPTOR_LOG_LEVEL : "info";
+const logFile = process.env.RAPTOR_LOG_LOCATION || "/var/log/raptor/raptor.log";
+const logToConsole = process.env.RAPTOR_LOG_TO_CONSOLE || false;
```

Comment:
Having just run into this, can we turn the console log on by default? The failure was completely silent as the console output nothing!

Reply:
209959d
--
Added.